praw/models/reddit/subreddit.py

```diff
@@ -1393,6 +1405,19 @@ def update(
             "text_color": text_color,
             "text_editable": text_editable,
         }
+        if fetch:
+            _existing_data = [
+                template
+                for template in iter(self)
+                if template["id"] == template_id
+            ]
+            if len(_existing_data) == 0:
+                raise InvalidFlairTemplateID(template_id)
+            else:
+                existing_data = _existing_data[0]
+                for key, item in data.copy().items():
+                    if not bool(item):
```

Comment:
This should be `if item is not None`. Otherwise the method cannot be used to set `False` or falsey (such as `""`) values.

Reply:
And I'd like there to be a tests that makes sure that some value (`text_editable`, maybe) can be set to `True` and *then* `False`, successfully.

Reply:
I implemented the first change, and I will generate tests for the second soon.